Makefile

```
@@ -8,27 +8,52 @@ GOARCH_aarch64 = arm64
 GOARCH_arm64 = arm64
 GOARCH ?= $(shell echo "$(GOARCH_$(ARCH))")
 
+VERSION := $(shell git describe --tags --always)
+REVISION := $(shell git rev-parse HEAD)
+PACKAGE := github.com/abiosoft/colima/config
+VERSION_VARIABLES := -X $(PACKAGE).appVersion=$(VERSION) -X $(PACKAGE).revision=$(REVISION)
+
+OUTPUT_DIR := _output/binaries
+OUTPUT_BIN := colima-$(OS)-$(ARCH)
+INSTALL_DIR := /usr/local/bin
+BIN_NAME := colima
+
+LDFLAGS := $(VERSION_VARIABLES)
+
+.PHONY: all
 all: build 
 
+.PHONY: clean
 clean:
 	rm -rf _output _build
 
+.PHONY: gopath
 gopath:
 	go get -v ./cmd/colima
 
+.PHONY: fmt
 fmt:
 	go fmt ./...
 	goimports -w .
 
+.PHONY: build
 build:
-	GOOS=$(GOOS) GOARCH=$(GOARCH) OS=$(OS) ARCH=$(ARCH) sh scripts/build.sh
+	go build -ldflags="$(LDFLAGS)" -o $(OUTPUT_DIR)/$(OUTPUT_BIN) ./cmd/colima
+	openssl sha256 -r -out $(OUTPUT_DIR)/$(OUTPUT_BIN).sha256sum $(OUTPUT_DIR)/$(OUTPUT_BIN)
+
+.PHONY: test
+test:
+	go test -race -v -ldflags="$(LD_FLAGS)" ./cmd/...
 
+.PHONY: vmnet
 vmnet:
 	sh scripts/build_vmnet.sh
 
+.PHONY: install
 install:
-	cp _output/binaries/colima-$(OS)-$(ARCH) /usr/local/bin/colima
-	chmod +x /usr/local/bin/colima
+	mkdir -p $(INSTALL_DIR)
```

Context: https://github.com/abiosoft/colima/pull/352/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R30
```suggestion
	mkdir -p $(INSTALL_DIR)
	rm -f $(INSTALL_DIR)/$(BIN_NAME)
```